### PR TITLE
Fix documentation pages

### LINKED ISSUE #4109: Documentation pages not loading assets correctly
When I redeployed the docs site just now (after merging #4080 which only touches `/docs/glossary.md`), pages within folders (including the /es/ pages) were being built with the wrong assets path so were being rendered without CSS.

Instead of the expected path prefix from `{{site.baseurl}}` of '/', it was trying to load from the local folder path e.g. `/docs/assets/css/...` and 404ing. To avoid the docs pages appearing broken, I've made the simplest possible fix of changing the asset paths in the layout file to start with a literal '/'.

My working assumption is that something has changed in how paths are evaluated and that merging the changes to the glossary page triggered a complete rebuild of the gh-pages site rather than the changes themselves having caused the error directly. (Full disclosure - I did not visit the page immediately before deploying but as @davewhiteland included screenshots I am reasonably confident in assuming the pages were working as expected pre deploy.)

It would be good to allocate some time to investigate this fully and possibly come up with a better fix. 

***Note:*** images are currently broken on the Spanish pages as these also rely on the `{{site.baseurl}}/assets/` syntax

### PR DESCRIPTION
**Pull request by lizconlan**
Closes #4109

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
 source 'https://rubygems.org'
-gem 'github-pages'
+gem 'github-pages', '148'
```
Comment (garethrees) [inline Gemfile]: I think we always want the latest of this, so don't need to pin.

File: _config.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 safe: true
 highlighter: rouge
-baseurl: /
+baseurl: ""
```
Comment (lizconlan) [inline _config.yml]: we also use "/es/" so it's probably safest to set the default

File: Gemfile (hunk 1 above)
Comment (lizconlan) [inline Gemfile]: the latest is beyond what GitHub have documented as current

File: _config.yml (hunk 2 above)
Comment (lizconlan) [inline _config.yml]: (I'll try it without, see what happens)

Comment (lizconlan) [inline _config.yml]: it does seem to work locally without this being set here, removing

**Pull request merged**